### ISSUE TITLE
Application will fail to open at times

### ISSUE THREAD

**Issue by gonzalezb** [libredeb/slingscold#3]
When you use it will open right away but if you close it a try to open it again you have to try 3 to 6 times it varies before it will open again.

**Logs from terminal:**

> Gtk-Message: 12:50:43.204: Failed to load module "appmenu-gtk-module"
> Gtk-Message: 12:50:45.419: Failed to load module "appmenu-gtk-module"
> **
> ERROR:arraylist.c:1181:gee_array_list_real_get: assertion failed: (index >= 0)
> Bail out! ERROR:arraylist.c:1181:gee_array_list_real_get: assertion failed: (index >= 0)
> Gtk-Message: 12:50:55.430: Failed to load module "appmenu-gtk-module"
> **
> ERROR:arraylist.c:1181:gee_array_list_real_get: assertion failed: (index >= 0)
> Bail out! ERROR:arraylist.c:1181:gee_array_list_real_get: assertion failed: (index >= 0)
> Gtk-Message: 12:50:56.651: Failed to load module "appmenu-gtk-module"
> **
> ERROR:arraylist.c:1181:gee_array_list_real_get: assertion failed: (index >= 0)
> Bail out! ERROR:arraylist.c:1181:gee_array_list_real_get: assertion failed: (index >= 0)
> Gtk-Message: 12:50:57.484: Failed to load module "appmenu-gtk-module"
> **
> ERROR:arraylist.c:1181:gee_array_list_real_get: assertion failed: (index >= 0)
> Bail out! ERROR:arraylist.c:1181:gee_array_list_real_get: assertion failed: (index >= 0)
> Gtk-Message: 12:50:58.312: Failed to load module "appmenu-gtk-module"**

**Comment by libredeb** [comment]
Ok, @gonzalezb I'm writing a new app cleaning all the old code... this one is called "LightPad" and I added more debug points... Maybe in a few weeks can release the first stable version. I can count on you to try this scenario again with the new app version? Thank for your support!

**Comment by gonzalezb** [comment]
Yes i will most definitely give it a go i use this as my main application launcher.

**Comment by gonzalezb** [comment]
@libredeb Yes i will most definitely give it a go i use this as my main application launcher.

**Comment by gonzalezb** [comment]
@libredeb This issue is fixed in lightpad thank you very much keep up to good work and ill continue to use this application thank you again.
